src/plugins/saved_objects_management/public/management_section/objects_table/saved_objects_table.tsx

```diff
@@ -178,43 +178,63 @@ export class SavedObjectsTable extends Component<SavedObjectsTableProps, SavedOb
   }
 
   fetchCounts = async () => {
-    const { allowedTypes } = this.props;
+    const { allowedTypes, namespaceRegistry } = this.props;
     const { queryText, visibleTypes, visibleNamespaces } = parseQuery(this.state.activeQuery);
 
     const filteredTypes = allowedTypes.filter(
       (type) => !visibleTypes || visibleTypes.includes(type)
     );
 
+    let availableNamespaces = namespaceRegistry.getAll() || [];
```

Comment:
I briefly looked into the idea of having a convention for `_all` namespaces to reduce the size of the body of the request, but it looks like this would require changes in `src/core/server/saved_objects/service/lib/repository.ts` (https://github.com/opensearch-project/OpenSearch-Dashboards/blob/main/src/core/server/saved_objects/service/lib/repository.ts#L722) and I wasn't able to determine what would be required to implement this.

@cliu123 Can we include all registered namespaces in the request or should we look into providing an `_all` request?

Reply:
No, we cannot. `availableNamespaces` provides by security backend plugin that evaluates the user's tenant permissions. Not all users have permissions to all tenants.